### DISCUSSION TITLE
Does DocTR support Arabic text recognition?

### DISCUSSION THREAD

**Discussion by khalilRhouma** [mindee/doctr#1434] [Q&A] [answered]
Hello,
First of all, thank you for your work on DocTR. I want to run the model on some Arabic document, but the model is not recognizing the Arabic language, it is detecting a Latin language instead. I saw that Arabic vocab does exist in your code, but I don't know if the model support Arabic recognition or not. Thanks in advance.

**Comment by felixdittrich92** [comment, marked as answer]
Hi @khalilRhouma :wave:,

We don't have any rrecognition model trained on the arabic vocab atm.
But if you have some data you can train your own model see: https://mindee.github.io/doctr/using_doctr/custom_models_training.html

Otherwise you could also create a synth dataset for example with: https://github.com/clovaai/synthtiger

Best regards,
Felix

**Reply by khalilRhouma** [reply]
Great, thank you @felixdittrich92
